src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -558,11 +558,12 @@
 
 helps['aks get-credentials'] = """
 type: command
-short-summary: Get access credentials for a managed Kubernetes cluster.
+short-summary: Get access credentials for a managed Kubernetes cluster. Merge them into .kube/config file so kubectl can use them
```

Comment:
merge only happens when -f is not specified. should we always put it in short description?

Reply:
Ah, yes, we should be clearer.  I'll make a small change to correct that.

Reply:
I've added a new commit, to remove that from short-description, and add the following long-description
>By default, the credentials are merged into the .kube/config file so kubectl can use them.  See -f parameter for details.